### ISSUE TITLE
The selected language in code block dropdown is not visible

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5#5859]
## 📝 Provide detailed reproduction steps (if any)

1. Open code block feature guide
2. Place caret in either JavaScript or HTML/XML block
3. Open code block dropdown

### ✔️ Expected result

A selected language is visible (is scrolled to if possible)

### ❌ Actual result

The active dropdown action is not visible.

## 📃 Other details

![Selection_075](https://user-images.githubusercontent.com/247363/69799859-609f1a00-11d4-11ea-9a2a-a21c270dc2b9.png)

* Browser: Safari, Firefox probably Chrome also
* CKEditor version: latest master

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by jodator** [comment]
It's worse on Safari as they don't render the scrollbar there and the list looks like finite:

![Zrzut ekranu 2019-11-28 11 55 25](https://user-images.githubusercontent.com/247363/69800925-7c0b2480-11d6-11ea-8f75-07be4c1bb392.png)

**Comment by bbenjamin** [comment]
Something to note, that may mean this is easier to solve: This is a problem on pages that have loaded with existing code blocks. If a new code block is added, or the language in an existing code block is changed, then the active language is automatically scrolled to when the select is opened. This can be reproduced on https://ckeditor.com/docs/ckeditor5/latest/features/code-blocks.html
